pkg/convert/tf.go

```diff
@@ -3880,9 +3939,51 @@ func sandboxedModulePackageBlock(
 	return block
 }
 
+func appendComment(body *hclwrite.Body, comment string) {
+	line := fmt.Sprintf("// %s\n", comment)
+	body.AppendUnstructuredTokens(hclwrite.Tokens{
+		{
+			Type:  hclsyntax.TokenIdent,
+			Bytes: []byte(line),
+		},
+	})
+}
+
+func localPulumiTerraformModulePackageBlock(
+	packageName string,
+	localPath string,
+) *hclwrite.Block {
+	block := hclwrite.NewBlock("package", []string{packageName})
+	body := block.Body()
+	body.SetAttributeValue("baseProviderName", cty.StringVal("terraform-module"))
+	body.SetAttributeValue("baseProviderVersion", cty.StringVal("0.1.4"))
+	paramBlock := hclwrite.NewBlock("parameterization", []string{})
+
+	// hardcoded version to align with the version in pulumi-terraform-module
```

Comment:
Ah interesting, thanks for this comment. Probably not worth build-depending on the pulumi-terraform-module code for this. Perhaps to make it more prominent we can lift it to a top-level const.